packages/provider-elasticsearch/src/example-types/results/highlightUtil.js

```diff
@@ -0,0 +1,71 @@
+import _ from 'lodash/fp.js'
+import F from 'futil'
+
+let highlightContentRegEx = ({ pre_tags, post_tags }) =>
+  new RegExp(`(?<=${pre_tags[0]})(.*?)(?=${post_tags[0]})`, 'g')
+
+let getHighlightMatches = _.curry((nodeHighlight, highlight) =>
+  highlight.match(highlightContentRegEx(nodeHighlight))
+)
+
+let getWordListMatchesRegEx = (words) => new RegExp(words.join('|'), 'gi')
+
+let highlightWith = ({ pre_tags, post_tags }) =>
+  F.highlight(pre_tags[0], post_tags[0])
+
+let findHighlightTagsRegEx = ({ pre_tags, post_tags }) =>
```

Comment:
This is the same regex as https://github.com/smartprocure/contexture/blob/main/packages/provider-elasticsearch/src/example-types/results/highlighting.js#L7